integration/unplugin/src/index.ts

```diff
@@ -1,5 +1,17 @@
-import { TransformResult, createUnplugin } from 'unplugin';
-import civet from '@danielx/civet';
+import {
+  TransformResult,
+  createUnplugin,
+  SourceMapCompact as UnpluginSourceMap,
+} from 'unplugin';
+import civet, { SourceMap } from '@danielx/civet';
+import {
+  remapRange,
+  flattenDiagnosticMessageText,
+  rangeFromTextSpan,
+  // @ts-ignore
```

Comment:
Will we need to publish this first so we can use it in the build step?

Reply:
I guess this is why the change above switched the build step from the unplugin esbuild to the OG esbuild plugin.

Reply:
`@danielx/civet` is marked as external for the plugin, so we the build step should work without needing to publish, but yeah to remove `@ts-ignore` we will need to publish this first

Reply:
👍